gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.0-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-all.zip
```

Comment:
According to @ysb33r this should stay at 2.0 so ew can ensure better Gradle 2.x compatibilty. If you look in our build.gradle, we use a compile time dependency of `gradleApi()` so this version does matter